database/migrations/2016_06_01_000001_create_oauth_auth_codes_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
het artisan:refresh commando dat gebruikt wordt bij het deployen en testen wilt standaard alleen de seeds en migrations pakken die in het project zelf staan. Er is een --from-path opdracht oid die hier mee zou helpen, maar die heb ik niet werkend gekregen. Ik heb bij de voorstel veranderingen voor het volgende team een entry toegevoegd om dat op te lossen. Hetzelfde geld voor delta_service spul